routes/routes.go

```diff
@@ -0,0 +1,30 @@
+package routes
+
+type InputRoute struct {
+	Name       string   `json:"name"`
+	Input      string   `json:"input"`
+	Outputs    []string `json:"outputs"`
+	Plugins    Plugins  `json:"plugins"`
+	Template   string   `json:"template"`
+	Scheduling chan struct{}
+}
+
+type Plugins struct {
+	AggregateIssuesNumber   int    `json:"aggregate-issues-number"`
+	AggregateIssuesTimeout  string `json:"aggregate-issues-timeout"`
+	PolicyShowAll           bool   `json:"policy-show-all"`
+	AggregateTimeoutSeconds int
+}
+
+func (route *InputRoute) IsSchedulerRun() bool {
+	return route.Scheduling != nil
```

Comment:
we stop `Scheduling` via close a channel. Will `route.Scheduling` be `nil` after 'StopScheduler'?

Reply:
no it won't. Route is not supposed to use after webhook termination